packages/cozy-client/src/models/index.js

```diff
@@ -1,6 +1,7 @@
-export { triggerResults } from './triggers'
+import * as triggers from './triggers'
 import * as instance from './instance'
 import * as applications from './applications'
 import * as file from './file'
+import * as accounts from './accounts'
```

Comment:
> Personally, I think that a Model should be singular since for instance we're dealing with only one account in this model. 

I agree, and the majority of other models only deal with a singular model also, they should be renamed.

```
rg --iglob '!*spec*'  '=>'
applications.js
12:export const getStoreInstallationURL = (appData = [], app = {}) => {
33:export const isInstalled = (apps = [], wantedApp = {}) => {
35:    app => app.attributes && app.attributes.slug === wantedApp.slug
44:export const getUrl = app => {
56:export const getAppDisplayName = (app, lang) => {

instance.js
6:export const isSelfHosted = instanceInfo => {
9:export const arePremiumLinksEnabled = instanceInfo => {
14:export const isFreemiumUser = instanceInfo => {
18:export const getUuid = instanceInfo => {
30:export const shouldDisplayOffers = data => {
47:export const hasAnOffer = data => {
61:export const buildPremiumLink = instanceInfo => {

triggers.js
4:  getLastExecution: triggerState => triggerState.current_state.last_execution,
6:  isErrored: triggerState => triggerState.current_state.status === 'errored'
10:  isKonnectorWorker: trigger => trigger.worker === 'konnector',
11:  getKonnector: trigger => {
```

Reply:
I agree that we need to be consistent. Since the doctypes they relate to are plural, I find plural makes sense… but i don't mind either way.

Reply:
Singular it is!